src/scb/wowi.clj

```diff
@@ -145,15 +145,16 @@
           anchor (-> snippet (select [[:a (html/attr-contains :href "fileinfo")]]) first)
           label (-> anchor :content first trim)
           truncated? (clojure.string/ends-with? label "...")
+          web-updated-date (-> snippet (select [:div.updated html/content]) first extract-updated-date)
           struct {:source :wowinterface
                   :source-id (extract-source-id anchor)
-                  :listing-name (-> label utils/slugify)
+                  :listing-name (-> label utils/slugify) ;; todo: rename `:name` and sort merge order
+                  :updated-date (format-wowinterface-dt web-updated-date)
                   :wowi/url (web-addon-url (extract-source-id anchor))
                   :wowi/listing-title label ;; I've seen underscores and truncation here, do not use if you can avoid it
                   ;; favourites? author? we can source these reliably from the API
-                  :wowi/web-updated-date (-> snippet (select [:div.updated html/content]) first extract-updated-date)
-                  :wowi/downloads (-> snippet (select [:div.downloads html/content]) first (clojure.string/replace #"\D*" "") utils/to-int)}
-          ]
+                  :wowi/web-updated-date web-updated-date
```

Comment:
revert to wowi/updated-date

Reply:
all these web-* prefixes need fixing. data is captured in it's own file now and the merge order is now sorted to prevent dodgy web data from making it into the final struct